lib/api/WebApi.js

```diff
@@ -22,8 +23,12 @@ async function deleteStudyRequestComment(csrf, studyRequest, comment) {
     method: 'DELETE',
     csrf,
   };
-  const result = await apiClient.fetch(url, options);
-  return SuccessResponse.validateAsync(result);
+  const response = await apiClient.fetch(url, options);
+  const responseSchema = Joi.object().keys({
```

Comment:
once our REST API here stabilizes, we should move these endpoint-specific schemas to a separate file that's imported in both `WebApi` and the relevant REST API controllers.